src/components/forms/PlanForm/helpers.ts

```diff
@@ -776,7 +777,15 @@ export function getGoalUnitFromActionCode(actionCode: PlanActionCodesType): Goal
  * @param {InterventionType} planType - plan type
  */
 export const isPlanTypeEnabled = (planType: InterventionType): boolean =>
-  ENABLED_PLAN_TYPES.includes(planType);
+  DISPLAYED_PLAN_TYPES.includes(planType);
+
+/**
+ * Check if plan type should be created
+ * @param {InterventionType} planType - plan type
+ * @param {boolean} isEditMode - are we editing or creating a plan
+ */
+export const shouldPlanTypeBeCreated = (planType: InterventionType, isEditMode: boolean): boolean =>
```

Comment:
Maybe we should rename this?  Seems like it is not accurate? i.e. you cannot create plans in edit mode.  So this means something other than "create"